.travis.yml

```diff
@@ -9,14 +9,18 @@ addons:
     packages:
       libicu-dev
 
-r_github_packages: jimhester/covr
+r_github_packages:
+  - jimhester/covr
+  - jimhester/lintr
```

Comment:
It might overcomplicate things to install lintr master when assessing lintr in a new PR. maybe just use the PR's copy of lintr during lining step.

Reply:
Oh, isn't that happening in line 22 anyway?
If that's the case, I'd revert `r_github_packages` and we're done?